src/main/java/org/folio/marccat/resources/domain/FixedField.java

```diff
@@ -1212,7 +1267,8 @@ public String getCartographicIndexAvailabilityCode() {
   }
 
   /**
-   * @param cartographicIndexAvailabilityCode The cartographicIndexAvailabilityCode
+   * @param cartographicIndexAvailabilityCode The
```

Comment:
Put it back on the same line for example:

    @param cartographicIndexAvailabilityCode The cartographicIndexAvailabilityCode

Reply:
It is solved in other PR